### PR TITLE
Allow empty services entry in compose file

### PR DESCRIPTION
**Pull request by CameronDiver**
Change-type: minor
Connects-to: #8
Signed-off-by: Cameron Diver <cameron@resin.io>
---- Autogenerated Waffleboard Connection: Connects to #8

### CODE REVIEW

File: src/compose.ts [hunk 1]
```diff
@@ -62,12 +62,10 @@ export function normalize(c: any): Composition {
...
 			const services: Dict<Service> = c.services;
 			const serviceNames = _.keys(services);
-			if (!_.size(services)) {
-				throw new ValidationError('Composition defines no services');
-			}
+
 			c.services = _.mapValues(services, (service) => {
 				return normalizeService(service, serviceNames);
-			});
+			}) || { };
```
Comment (dfunckt) [inline src/compose.ts]: Why is that needed? If it can be undefined, let's set it above, at `const services = c.services || {}`.

Comment (CameronDiver) [reply src/compose.ts]: I guess it does make more sense there, you're right.
